sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -93,6 +96,22 @@ public DefaultAzureCredentialBuilder managedIdentityClientId(String clientId) {
         return this;
     }
 
+    /**
+     * Specifies the resource ID of user assigned or system assigned identity, when this credential is running
+     * in an environment with managed identities. If unset, the value in the AZURE_CLIENT_ID environment variable
+     * will be used. If neither is set, the default value is null and will only work with system assigned
+     * managed identities and not user assigned managed identities.
+     *
+     * Only one of managedIdentityResourceId and managedIdentityClientId can be specified.
```

Comment:
nit: Javadocs are effectively HTML, so the empty line here will be removed. You'll want to add a `<p>` to actually separate these lines

Reply:
Ah, thanks. I copied the comment block from above. I'll leave it since it's consistent. Good tip for the future!